e2e/armada-operator.patch

```diff
@@ -13,6 +22,16 @@ index d76880c..3f0a159 100644
  ---
  apiVersion: install.armadaproject.io/v1alpha1
  kind: Lookout
+@@ -156,6 +162,9 @@ spec:
+         password: psw
+         dbname: scheduler
+         sslmode: disable
++    scheduling:
++      indexedNodeLabels:
++        - armada-spark
```

Comment:
How does this effect armada?  is it something that GR would be willing to set in  their prod clusters?  does it need to be unique per user, or do they only need to do it once for all users?

Reply:
The scheduler *must* be configured to index labels related to gang scheduling or it won't work. This is based on my reading of armada scheduler source and testing of armada locally.